### ISSUE TITLE
Rename acme-blockchain to antelope throughout the code

### ISSUE THREAD

**Issue by matthewdarwin** [EOS-Nation/firehose-antelope#25]
See error message:

```
2022-11-26T16:36:02.826Z ERRO (derr) acme-blockchain {"error": "unable to launch: unable to create app \"merger\": failed to create directory \"{sf-data-dir}/storage/forked-blocks\": mkdir {sf-data-dir}: permission denied"}
```

**Comment by matthewdarwin** [comment]
Another example:

```
/usr/bin/fireantelope -d /var/lib/dfuse -c /etc/dfuse/config.yml
Acme on StreamingFast
```

**Comment by shaqk** [comment]
These need a global search and replace for the three casing varients of acme mentions

**Comment by fschoell** [comment]
this has been fixed in #88
